menu/menu.yaml

```
@@ -1071,6 +1071,8 @@
       Title: Failed Messages
     - Url: servicepulse/intro-failed-message-retries
       Title: Retrying Failed Messages
+    - Url: servicepulse/intro-editing-messages
+      Title: Editing Failed Messages
```

```suggestion
      Title: Fixing malformed messages
```

Outcome of a discussion during a DevExp sync.